net/v2ray/Portfile

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+name                v2ray
+version             4.21.3
+categories          net security
+platforms           darwin
+license             MIT
+maintainers         {@casouri gmail.com:casouri} openmaintainer
+description         A platform for building proxies to bypass network restrictions
+
+long_description    Project V is a set of network tools that help you to build your own computer network. \
+                    It secures your network connections and thus protects your privacy.
```

Comment:
This description doesn't seem terribly good...

Reply:
I copied it from the homepage. In what aspect do you think it can be improved?

Reply:
It's incomprehensible. I have no idea what sort of tool this is from the long description. Write something short in your own words that would inform a person browsing our portfiles why they want to install this.

Reply:
How is the new one? I tried to summarize in my words.

Reply:
Sorry, I thought I pushed my change... Here is the update.